src/App.scss

```diff
@@ -41,6 +42,9 @@ h5 {
 
 .main {
     width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
The design looks really great 💯 . I have some thoughts below about how you can make the survey width consistent (not always changing based on its content) - just something to consider. Let me know if I'm misunderstanding the intention, and of course there could be other ways than what I'm suggesting

---

Using `align-items: center` in a flex column will make the items shrink to the width of its inner content, in this case I'm seeing:
<img width="895" alt="Screen Shot 2021-01-27 at 5 17 32 PM" src="https://user-images.githubusercontent.com/10643091/106071559-cda58c80-60c3-11eb-8e68-a2ac3357183a.png">

But if you remove those flex properties (lines 45-47), it will keep a consistent width according to the width and media queries that have been defined, and look like this:
<img width="886" alt="Screen Shot 2021-01-27 at 5 31 41 PM" src="https://user-images.githubusercontent.com/10643091/106072490-86b89680-60c5-11eb-922e-33bfb3430bee.png">

Then, to center the Reset button (assuming that was the intention with the center aligned flex column), you could wrap the button in a div and style that to center its contents
```
<div style="text-align: center">
    <button class="btn-nav">Reset</button>
</div>
```